components/blocks/cards/LogInCard/LogInCard.vue

```diff
@@ -1,3 +1,67 @@
+<script setup lang="ts">
+import {
+  defineComponent,
+  reactive,
+  toRefs,
+  useContext,
+} from '@nuxtjs/composition-api';
+import HideShowPassword from '@/components/elements/buttons/HideShowPassword/HideShowPassword.vue';
+import BaseInput from '@/components/elements/BaseInput.vue';
+import Card from '@/components/elements/cards/Card.vue';
+import CardBody from '@/components/elements/cards/CardBody.vue';
+import CardHeader from '@/components/elements/cards/CardHeader.vue';
+import BaseButton from '@/components/elements/buttons/BaseButton.vue';
+import CloseButton from '@/components/elements/buttons/CloseButton.vue';
+
+export default defineComponent({
```

Comment:
This wouldn't be valid under <script setup>

Reply:
Oh shit. Thought I reverted that one. Was working on it but didn’t finish.

Reply:
I reverted it back. Not that comfortable with Vue3 to revamp the LogInCard without being able to actually build it and see if it works 🙃